methods.tex

```diff
@@ -260,7 +255,32 @@ \subsection{French \gls{SFR} Deployment Schedule}
 material flow, and demonstrates that, if such an aggressive deployment scheme 
 took place, the \glspl{SFR} would have enough fuel.  More realistically, the 
 deployment of new \glspl{SFR} can be spread out by staggering scheduled 
-decommissioning of \glspl{LWR} through lifetime extensions.  An analysis of the 
+decommissioning of \glspl{LWR} through lifetime extensions. For example,
+we increased the original lifetime of French \glspl{LWR} (60 years) randomly 
+by sampling from a uniform distribution from zero to 25 years. 
```

Comment:
"from zero to 25 years. " -> "of lifetime extension magnitudes between 0 and 25 years."